tests/test_mapping/test_mapping_helpers.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import pandas as pd
+import numpy as np
+
+# Local Imports
+from src.mapping.mapping_helpers import update_ref_list
+
+
+class TestUpdateRefList(object):
+    """Tests for update_ref_list."""
+
+    @pytest.fixture(scope="function")
+    def full_input_df(self):
+        """Main input data for update_ref_list tests."""
+        columns = ["reference", "instance", "formtype", "cellnumber"]
+        data = [
+            [49900001031, 0.0, 6, 674],
+            [49900001530, 0.0, 6, 805],
+            [49900001601, 0.0, 1, 117],
+            [49900001601, 1.0, 1, 117],
+            [49900003099, 0.0, 6, 41],
+        ]
+        df = pd.DataFrame(columns=columns, data=data)
+        df["formtype"] = df["formtype"].apply(lambda x: str(x))
```

Comment:
does it work to do df = df.astype({"formtype":int}) or something like that?

Reply:
I think maybe for the test we should have our formtype values as "0006" and "0001" as they are in the full responses.

Reply:
I've changed instance to an int - but formtype is handled as a string